dev-lisp/sbcl/sbcl-2.5.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+portVersion="2.5.0"
+SOURCE_URI="https://github.com/sbcl/sbcl/archive/refs/tags/sbcl-$portVersion.zip"
+CHECKSUM_SHA256="15fc66e96683ae76ebfbb2e8b7d2d9fb443358ada2813fba860e65f4859346d6"
+SOURCE_DIR="sbcl-sbcl-$portVersion"
+PATCHES="sbcl-2.5.0.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	sbcl$secondaryArchSuffix = $portVersion
+	cmd:sbcl$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libzstd$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:clisp
+	cmd:gcc$secondaryArchPrefix
+	cmd:git
+	cmd:make
+	cmd:rm
+	cmd:sh
+	"
+
+BUILD()
+{
+	chmod a+x *.sh
+	sh make.sh --xc-host='clisp -norc' --with-sb-core-compression --with-sb-simd-pack --with-sb-thread
+}
+
+INSTALL()
+{
+	mkdir -p $manDir
+	mkdir -p $docDir
+	INSTALL_ROOT=$prefix MAN_DIR=$manDir DOC_DIR=$docDir INFO_DIR=$infoDir sh install.sh
+}
+
+# Fails at this stage
+#TEST()
+#{
+#    cd tests
+#    ./run-tests.sh
+#}
```

Comment:
You can leave this enabled. The tests aren't run automatically.

Reply:
Thanks. For now, it is a reminder for me to cleanup tests.